tests/test_informationmodel.py

```diff
@@ -1560,3 +1560,29 @@ def test_to_graph_should_return_standard() -> None:
     g2 = Graph().parse(data=src, format="turtle")
 
     assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_conforms_to() -> None:
+    """It returns an conforms_to graph isomorphic to spec."""
+    informationmodel = InformationModel("http://example.com/informationmodels/1")
+    standard = Standard("http://example.com/standards/1")
+    informationmodel.conforms_to = standard
+
+    src = """
```

Comment:
Vi bør ikkje lage eit "ekstra" trippel når det er kun ei lenke som er satt i instansen av Standard. Så src i testen skal vere som følger:
```
    src = """
    @prefix dct: <http://purl.org/dc/terms/> .
    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
    @prefix dcat: <http://www.w3.org/ns/dcat#> .
    @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
    @prefix prof: <http://www.w3.org/ns/dx/prof/> .
    <http://example.com/informationmodels/1> a modelldcatno:InformationModel ;
        dct:conformsTo <http://example.com/standards/1>
    .
    """
```

Reply:
Hei @stigbd , dette har vi jo allerede snakket om og det er satt opp en større task på å endre på dette. Foreløpig har jeg ikke det helt klart for meg hvordan det vi snakket om skal implementeres dvs at dersom et objekt ikke har noen satte object properties eller data properties, så skal det ikke "tas med" i grafen, kun lenken Men som systemutviklingsprinsipp og for ordens skyld er det kanskje greit å skille disse to taskene fra hverandre og ikke gjøre begge to tingene samtidig når det er såpass mange steder som må endres. Et annet spørsmål som jeg også sitter og lurer på nå som jeg ser det, det opprettes jo et objekt, så det da virker det jo litt rart at det ikke skal serialiseres som et selvstendig objekt? Er dette virkelig riktig? Tidligere ble vi enige om at vi skulle kunne sette lenker i stedet for objekter, men her opprettes det jo et i objekt av typen Standard i testen. Så nå ble jeg plutselig veldig usikker.

Men dersom det er viktigere å få gjort gjort disse endringene så kan jeg legge fra meg modelldcatno-1.1-ting og ta den tasken i stedet. Det er bare at jeg vegrer meg for å gjøre to ting samtidig, bedre å gjøre det i tur og orden, spesielt når jeg ikke har det helt klart for meg hvordan det skal implementeres.

#114

Reply:
Har blitt enig med @shenluna og de andre på sprintplanleggingen i dag at vi tar den saken senere når jeg er ferdig med 1-1-tingene. Dette også fordi 1-1-tingene er ting som potensielt skal brukes i frontend og dette er ikke noen åpenbar showstopper, slik jeg oppfatter, dette har jo vært slik en god stund allerede . Men dersom dere har flere innspill så kan vi ta et lite møte om det, selvfølgelig.

Reply:
> Tidligere ble vi enige om at vi skulle kunne sette lenker i stedet for objekter, men her opprettes det jo et i objekt av typen Standard i testen. Så nå ble jeg plutselig veldig usikker.

Enig i at dette kan virke litt uryddig. Men bruker av biblioteket vil ha mulighet til _kun_ å oppgi lenke (som i Linked Data) til eit anna (remote) triple. Før fiksa vi dette ved å gi property en lenke som `str`. Men etter at vi innførte dei nye init (constructors) endra eg litt i implementasjonen.

Eg er uansett enig i at vi kan ta dette i eit anna issue.

Reply:
Flott @stigbd ! Jeg er også enig i at det kan være nyttig å ha mulig til kun å oppgi lenke. Men vi tar en skikkelig omgang på dette når jeg går i gang med den oppgaven slik at jeg er helt sikker på hvordan dere vil ha. Jeg skal det etter disse modelldcatno-1-1-tingene. Da merger jeg nå.